projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -97,6 +100,19 @@ public static function load_wpcom_rest_api_endpoints() {
 		}
 	}
 
+	/**
+	 * Adds a global variable containing the map provider in a map_block_settings object to the window object
+	 *
+	 * @return void
+	 */
+	public static function load_map_block_settings() {
+		$map_provider = apply_filters( 'wpcom_map_block_map_provider', 'mapbox' );
+
+		wp_register_script( 'map-block-settings', '', array(), '1.0', true );
+		wp_localize_script( 'map-block-settings', 'map_block_settings', array( 'provider' => $map_provider ) );
+		wp_enqueue_script( 'map-block-settings', '', array(), '1.0', true );
```

Comment:
Instead of registering a new script, what do you think about hooking into the existing Jetpack block bundle, where the map block is being enqueued? You can use this as an example:

https://github.com/Automattic/jetpack/blob/fc41f14311eff83abe2188c4aa6e6307287f3113/projects/plugins/jetpack/extensions/blocks/anchor-fm/anchor-fm.php#L200

Reply:
Thanks for pointing this out! I made the change.